go/tlsigning_test.go

```diff
@@ -102,6 +102,27 @@ func TestVerifyStaticSignature(t *testing.T) {
 	assert.Nilf(err, "signature verification should not fail: %v", err)
 }
 
+func TestVerify_WithInvalidSignature_ShouldReturnError(t *testing.T) {
```

Comment:
The name of this function should be `TestVerifyWithInvalidSignatureShouldReturnError`